python/cuml/test/dask/test_datasets.py

```diff
@@ -76,3 +76,55 @@ def test_make_blobs(nrows,
 
     finally:
         c.close()
+
+
+@pytest.mark.parametrize('n_samples', [100, 100000])
```

Comment:
It would be nice if you do the parameters using `unit_` and `stress_params` like the usual tests: https://github.com/rapidsai/cuml/blob/a18ea88f8d73dc24d0310087daf871a3baaf584d/python/cuml/test/test_kmeans.py#L37 BTW I don't think `quality_param` is needed, but stress probably is to avoid finding issues with bigger datasets